src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -58,11 +58,12 @@ public class FrameworkController : MarshalByRefObject
         /// <summary>
         /// Construct a FrameworkController using the default builder and runner.
         /// </summary>
-        public FrameworkController(string assemblyPath, IDictionary settings)
+        public FrameworkController(string assemblyPath, string idPrefix, IDictionary settings)
```

Comment:
It wasn't done before, but this is part of the public API, so the parameters should be in the XMLDOC.